### PR TITLE
Improve CallTile UI

### PR DESCRIPTION
**Pull request by bwindels**
Mostly done but can be merged anyway so `bwindels/calls` contains the main improvements while I'm out tomorrow. Missing so far:
 - only include 5 or so member avatars?
 - emit updates from view model when member count changes
 - minor style changes like decide on alignment of tile (e.g. left instead of centered)

image
image

### CODE REVIEW

File: src/matrix/calls/CallHandler.ts [hunk 1]
```diff
@@ -130,12 +130,12 @@ export class CallHandler implements RoomStateHandler {
...
     }
 
-    createCall(roomId: string, type: "m.video" | "m.voice", name: string, intent?: CallIntent, log?: ILogItem): Promise<GroupCall> {
+    createCall(roomId: string, type: CallType, name: string, intent?: CallIntent, log?: ILogItem): Promise<GroupCall> {
         return this.options.logger.wrapOrRun(log, "CallHandler.createCall", async log => {
             if (!intent) {
                 intent = CallIntent.Ring;
             }
-            const call = new GroupCall(makeId("conf-"), true, { 
+            const call = new GroupCall(makeId("conf-"), true, undefined, {
```
Comment (MidhunSureshR) [inline src/matrix/calls/CallHandler.ts]: Why do we pass undefined as start time?

File: src/platform/web/dom/TimeFormatter.ts [hunk 2]
```diff
@@ -75,6 +79,37 @@ export class TimeFormatter implements ITimeFormatter {
...
+
+    formatDuration(milliseconds: number): string {
+        let days = 0;
+        let hours = 0;
+        let minutes = 0;
+        if (milliseconds > DAYS_IN_MS) {
+            days = Math.floor(milliseconds / DAYS_IN_MS);
+            milliseconds -= days * DAYS_IN_MS;
+        }
+        if (milliseconds > HOURS_IN_MS) {
```
Comment (MidhunSureshR) [inline src/platform/web/dom/TimeFormatter.ts]: Should the `>` be `>=` ?
Currently `formatDuration(25 hours in ms)` gives `1d 0h 60m 0s`

**Review by MidhunSureshR — APPROVED** [review]
Looks good, just have one question!

File: src/matrix/calls/CallHandler.ts (hunk 1 above)
Comment (bwindels) [reply src/matrix/calls/CallHandler.ts]: Because the call hasn't started yet. It is started by the remote echo of the call event we're about to send.

**Pull request merged**